wp-plugin-base/Dockerfile

```diff
@@ -0,0 +1,69 @@
+FROM php:7.1-apache
+MAINTAINER Thrijith Thankachan <thrijith13@gmail.com>
```

Comment:
Please use the LABEL. [Example Dockerfile](https://github.com/rtCamp/dockerfiles/blob/ab47e17b24fc0dabbd0dbd88b0b6ee6b47a911b3/nginx-build/Dockerfile#L3)